config/datadog_cik8s.yaml

```diff
@@ -11,3 +11,9 @@ agents:
       operator: "Equal"
       value: "true"
       effect: "NoSchedule"
+  image:
+    # official image without custom checks
+    repository: "datadog/agent"
+    tag: "7"
+    # we don't have to skip version test as we're not using the docker image digest
+    doNotCheckTag: false
```

Comment:
Is there a reason to not pick the default values from the chart, aka. https://github.com/DataDog/helm-charts/blob/93c4fe5f2e58537a25626bee9656a21ef30415d8/charts/datadog/values.yaml#L1191-L1212 ?